tide_site.module

```diff
@@ -576,27 +580,45 @@ function tide_site_taxonomy_term_delete($term) {
 /**
  * Implements hook_ENTITY_TYPE_update().
  */
-function tide_site_node_update($node) {
-  // Regenerate site aliases of the updated node.
-  /** @var \Drupal\tide_site\AliasStorage $alias_storage */
+function tide_site_node_update(NodeInterface $node) {
+  // Gets required services.
   $alias_storage_helper = \Drupal::service('tide_site.alias_storage_helper');
-
+  $tide_site_helper = \Drupal::service('tide_site.helper');
   $path_storage = \Drupal::entityTypeManager()->getStorage('path_alias');
 
-  /** @var \Drupal\node\NodeInterface $node */
+  // If the node has no aliases, we should creat them.
```

Comment:
```suggestion
  // If the node has no aliases, we should create them.
```

Reply:
Thanks @GROwen for your `Commit suggestion`.